CMakeLists.txt

```diff
@@ -5,6 +5,18 @@
 
 cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
 
+option(UNITTESTS "Run unit tests only." OFF)
+
+if(UNITTESTS)
+
+enable_testing()
+
+add_subdirectory(UNITTESTS)
+add_subdirectory(drivers/tests/UNITTESTS)
+add_subdirectory(connectivity/cellular/tests/UNITTESTS)
```

Comment:
we list here all unittests in Mbed OS tree. Shouldn't they be `EXCLUDE_FROM_ALL` to be not available if they are not requested ? I assume this is intentionally this way to build all tests?

Reply:
"EXCLUDE_FROM_ALL" exclude their source building under one project